src/main/java/me/magnet/consultant/ConfigUpdater.java

```diff
@@ -76,21 +81,26 @@ public String getValue() {
 
 	@Override
 	public void run() {
+		if (shutdownBegun.get()) {
+			log.info("Marked for shutdown. Not running");
```

Comment:
This can be a bit more descriptive: "Not retrieving new config since we're shutting down" for instance.